files/standalone/simp_standalone_finalize.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+
+usage="Usage: $0 -k (true|false*) -d (true|false*) -f (true*|false) -p (true*|false) [-h] [-d]"
```

Comment:
Program fails if you don't specify a value for -d. So you need to either fix the  getopts string or the usage